javascript/ql/src/semmle/javascript/ApiGraphs.qll

```diff
@@ -374,15 +374,13 @@ module API {
             exists(SSA::implicitInit([nm.getModuleVariable(), nm.getExportsVariable()]))
           )
         )
-        or
-        m = any(CanonicalName n | isDefined(n)).getExternalModuleName()
       } or
       MkModuleImport(string m) {
         imports(_, m)
         or
-        m = any(CanonicalName n | isUsed(n)).getExternalModuleName()
-        or
         any(TypeAnnotation n).hasQualifiedName(m, _)
+        or
+        any(Type t).hasUnderlyingType(m, _)
```

Comment:
Could you add a regression test that demonstrates the need for this addition?

Reply:
Hmmm. I can't find a situation where that line is needed. The `imports(_, m)` seems to cover all the cases I can come up with.  
I still think it's a good idea to have, to have consistency with the `MkHasUnderlyingType` implementation.

Reply:
:+1: